sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -909,7 +909,7 @@ public HttpPipeline getHttpPipeline() {
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Void> deleteModel(String modelId) {
         return deleteModelWithResponse(modelId)
-            .map(Response::getValue);
+            .flatMap(voidResponse -> Mono.empty());
```

Comment:
`.map(Response::getValue)` returns null, which is not a valid return item from a Mono. So we need to map it to `Mono.empty()` instead.